docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -552,10 +552,10 @@ Perform the following steps to prepare your source-replica database:
     --upgraded to ybvoyager_import_data_batches_metainfo_v3 post v1.6
     CREATE TABLE ybvoyager_metadata.ybvoyager_import_data_batches_metainfo_v3 (
                migration_uuid VARCHAR2(36),
-               data_file_name VARCHAR2(250),
+               data_file_name VARCHAR2(1000),
```

Comment:
1000 -> 4000 for all data type changes in this file.

Reply:
```suggestion
               data_file_name VARCHAR2(4000),
```